### PR TITLE
Add infra kuttl

### PR DESCRIPTION
**Pull request by vhariria**

### CODE REVIEW

File: kuttl-test.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-infra
+namespace: openstack
+timeout: 180
+parallel: 1
+suppress:
+  - events                     # Remove spammy event logs
+  -
```
Comment (abays) [inline kuttl-test.yaml:26]: Looks like an extra `-` here

File: tests/kuttl/tests/infra_scale/01-assert.yaml
Comment (abays) [inline tests/kuttl/tests/infra_scale/01-assert.yaml:1]: It doesn't appear that there is any actual scaling done in the `infra_scale` test

Comment (vhariria) [reply tests/kuttl/tests/infra_scale/01-assert.yaml:1]: This is a placeholder for future scale-tests.

File: tests/kuttl/common/assert_sample_deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#
```
[the remark this answers is not in the document]
Comment (abays) [reply tests/kuttl/common/assert_sample_deployment.yaml]: Without this and with everything else remaining as-is, the test now fails due to timeout.

[the remark this answers is not in the document]
Comment (abays) [reply tests/kuttl/common/assert_sample_deployment.yaml]: My issue could be a false alarm with my environment, however

[the remark this answers is not in the document]
Comment (abays) [reply tests/kuttl/common/assert_sample_deployment.yaml]: It was my environment.  Works for me now.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**